addons/event/models/event_event.py

```diff
@@ -671,6 +671,17 @@ def _get_ics_file(self):
             result[event.id] = cal.serialize().encode('utf-8')
         return result
 
+    def _get_tickets_dl_access_hash(self, registration_ids):
+        """ Returns the ground truth hash for accessing the tickets in route /event/tickets_dl.
+        The dl links are always made event-dependant, hence the method linked to the record in self.
+        """
+        self.ensure_one()
+        registration_ids.sort()
```

Comment:
this will modify inplace the argument received, can you do `sorted(registration_ids)` instead ? (so it create a new object and do not modify the one we received)